src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -589,6 +608,32 @@ public sealed interface MemorySegment permits AbstractMemorySegmentImpl {
      */
     long byteSize();
 
+    /**
+     * {@return the <em>maximum</em> byte alignment (which is equal to or higher than
+     * the requested byte alignment during native segment allocation)}
+     * <p>
+     * The returned alignment is always an even power of two and is derived from:
```

Comment:
Not sure about the "even". Surely 2^3 is a valid max alignment?

Reply:
The meaning of "even" here is that it is a number 2^n, n > 0, n ∈ Z. But maybe "a power of two" is enough.

Reply:
power of two implies even, so I'd say it's better to omit